pkg/keystone/schema.go

```diff
@@ -0,0 +1,35 @@
+package keystone
+
+import (
+	comv1 "github.com/openstack-k8s-operators/keystone-operator/pkg/apis/keystone/v1"
+	util "github.com/openstack-k8s-operators/lib-common/pkg/util"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"io"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"strings"
+)
+
+type schemaOptions struct {
+	DatabaseHostname string
+	SchemaName       string
+	SchemaPassword   string
+}
+
+// SchemaObject func
+func SchemaObject(cr *comv1.KeystoneAPI) unstructured.Unstructured {
```

Comment:
This function can generate errors, so its return **must** include an error. Panic isn't acceptable. We don't want to be trying to unwind these later. Lets get this right first time.